code/modules/client/preferences.dm

```diff
@@ -1618,9 +1691,9 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	character.h_style = h_style
 	character.f_style = f_style
 
+	character.rparts_data = rlimb_data //Passes organ data to the character's own variable so decisions can be made based on it later.

```

Comment:
you do realize any place this is used you could just use M.client.prefs.rlimb_data right